### ISSUE TITLE
telnet support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dragonlinux/connectbot#123]
```
Please add support for telnet to configurable ports. This would be great for 
connecting to 
nethack.alt.org directly :)

```

Original issue reported on code.google.com by `tau...@gmail.com` on 28 Feb 2009 at 2:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `kenny@the-b.org` on 5 Mar 2009 at 9:31
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I agree I'd really like telnet support in ConnectBot. I have another Telnet 
client
for Android but it doesn't have the great features that ConnectBot has.
```

Original comment by `Banniste...@gmail.com` on 3 Apr 2009 at 11:49

**Comment by GoogleCodeExporter** [comment]
```
With the terminal emulation ConnectBot has it would go beyond other terminal 
apps for 
Android so, yes! - this would be a great feature to have.

```

Original comment by `shonzi...@gmail.com` on 18 Apr 2009 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
This is almost done.
```

Original comment by `kenny@the-b.org` on 5 May 2009 at 10:11
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
This was done in r298 and is released to market as the 1.5 branch
```

Original comment by `kenny@the-b.org` on 30 Jun 2009 at 9:16
- Changed state: **Done**
